### PR TITLE
Add Cortana in Teams Pshell topic files

### PR DESCRIPTION
**Pull request by akshbhat-msft**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsTeamsCortanaPolicy.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+## DESCRIPTION
+
+The CsTeamsCortanaPolicy cmdlets enable administrators to control settings for Cortana voice assistant in Microsoft Teams. Specifically, these specify if a user can use Cortana voice assistant in Microsoft Teams and Cortana invocation behavior via CortanaVoiceInvocationMode parameter -
+* Disabled - Cortana voice assistant is disabled
+* PushToTalkUserOverride - Cortana voice assistant is enabled but without wake-word ("Hey Cortana") invocation
+* WakeWordPushToTalkUserOverride - Cortana voice assistant is enabled with wake-word ("Hey Cortana") invocation on devices where wake-word is supported
+
+## EXAMPLES
+
+### Example 1
```
Comment (tseward) [inline skype/skype-ps/skype/Get-CsTeamsCortanaPolicy.md:39]: Can you provide an example of using the -Filter parameter?

File: skype/skype-ps/skype/Get-CsTeamsCortanaPolicy.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+### Filter
+```
+Get-CsTeamsCortanaPolicy [-Tenant <System.Guid>] [-Filter <String>] [-LocalStore] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The CsTeamsCortanaPolicy cmdlets enable administrators to control settings for Cortana voice assistant in Microsoft Teams. Specifically, these specify if a user can use Cortana voice assistant in Microsoft Teams and Cortana invocation behavior via CortanaVoiceInvocationMode parameter -
```
Comment (get-itips) [inline skype/skype-ps/skype/Get-CsTeamsCortanaPolicy.md:32]:
```suggestion
The CsTeamsCortanaPolicy cmdlets enable administrators to control settings for Cortana voice assistant in Microsoft Teams. Specifically, if a user can use Cortana voice assistant in Microsoft Teams and determines Cortana invocation behavior via CortanaVoiceInvocationMode parameter -

```

File: skype/skype-ps/skype/Grant-CsTeamsCortanaPolicy.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DomainController
+
```
Comment (dariomws) [inline skype/skype-ps/skype/Grant-CsTeamsCortanaPolicy.md:68]:
```suggestion

This parameter is reserved for internal Microsoft use.

```

File: skype/skype-ps/skype/Grant-CsTeamsCortanaPolicy.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Global
+
```
Comment (dariomws) [inline skype/skype-ps/skype/Grant-CsTeamsCortanaPolicy.md:82]:
```suggestion

This parameter is reserved for internal Microsoft use.

```

File: skype/skype-ps/skype/Grant-CsTeamsCortanaPolicy.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### -PassThru
+
```
Comment (dariomws) [inline skype/skype-ps/skype/Grant-CsTeamsCortanaPolicy.md:111]:
```suggestion

This parameter is reserved for internal Microsoft use.

```

**Comment by akshbhat-msft** [comment]
> Just small suggestions.
> do you have a release date for these cmdlets? @akshbhat-msft Thanks

@get-itips  - Targeting end of this week (July 31st) to release. Latest Aug 4.

**Comment by scanum** [comment]
@chrisda Hi Chris! Hope you are doing well. I see that Dario, Andres, and Trevor have approved changes suggested by @akshbhat-msft. Wondering if this can be merged? This is an urgent request! Thank you.

**Comment by scanum** [comment]
@officedocsbot assign @chrisda

**Pull request merged**